appveyor.yml

```diff
@@ -21,10 +21,17 @@ environment:
 
   matrix:
   - TARGET: x86_64-pc-windows-msvc
+    CHANNEL: stable
   - TARGET: i686-pc-windows-msvc
+    CHANNEL: stable
+  # TODO: replace with UWP targets in future
```

Comment:
Is there no way to just check if the crate will build on UWP? Like we do with Fuchsia, Redox, UEFI, etc...

Reply:
You'd have to build your own libstd for this target using Xargo because there are no binaries for it.
I have no idea how much effort it would require.

Reply:
We [currently use `xbuild`](https://github.com/rust-random/getrandom/blob/98934cb4ab9438a9fec76fa22c0f3154890c2bc6/.travis.yml#L137-L139) to work with targets not yet shipping with `rustup`.

Running `cargo xbuild --target=x86_64-uwp-windows-gnu` works on my system, so we should (at the very least) add that and `i686-uwp-windows-gnu` to the travis script.